packages/comms/package.json

```diff
@@ -69,10 +69,10 @@
         "tmp": "0.2.1"
     },
     "devDependencies": {
-        "@hpcc-js/bundle": "^2.11.3",
+        "@hpcc-js/bundle": "^2.10.23",
         "@types/d3-request": "1.0.6",
         "@types/d3-time-format": "2.3.1",
-        "@types/node": "^10",
+        "@types/node": "^14",
```

Comment:
I don't believe this caused any problems, as I was able to build without warnings or errors. It was a required for a change in the "fs-extra" dependency, which was causing any of the "wsdl-..." npm scripts to fail

Reply:
"@types/node": "^16" would match what we are actually using?

Reply:
Looks like the CI failed because your package-lock file is out of sync (do a npm install and commit).